src/tqec/detectors/operation.py

```diff
@@ -0,0 +1,244 @@
+from typing import Any
+from dataclasses import dataclass
+
+import cirq
+
+STIM_TAG = "STIM_OPERATION"
+
+
+class ShiftCoords(cirq.Operation):
+    def __init__(self, *shift: int) -> None:
+        """
+        Annotates that the qubit/detector coordinate origin is being moved.
+
+        This is a replication of the [stimcirq.ShiftCoordsAnnotation](https://github.com/quantumlib/Stim/blob/main/glue/cirq/stimcirq/_shift_coords_annotation.py)
```

Comment:
Maybe
```suggestion
        This is a replication of the 
        [stimcirq.ShiftCoordsAnnotation](https://github.com/quantumlib/Stim/blob/main/glue/cirq/stimcirq/_shift_coords_annotation.py)
```
to shorten as much as possible the line?